cordova-lib/spec-plugman/fetch.spec.js

```diff
@@ -214,20 +214,23 @@ describe('fetch', function() {
         it('Test 017 : should fetch from a commit-sha', function(done) {
             wrapper(fetch('http://github.com/apache/cordova-plugin-device.git#ad5f1e7bfd05ef98c01df549a0fa98036a5625db', temp, { expected_id: 'cordova-plugin-device' }), done, function() {
                 expect(1).toBe(1);
+                done();
             });
-        });
+        }, 20000);
```

Comment:
3 occurrences of 20000 as a magic number here should be replaced by a constant, like save.spec.js which uses TIMEOUT and BIG_TIMEOUT